src/test/java/com/automationexercise/tests/Tests/AddtocartTest.java

```diff
@@ -0,0 +1,53 @@
+package com.automationexercise.tests.Tests;
+
+import com.automationexercise.DriverManager.DriverFactory;
+import com.automationexercise.DriverManager.DriverManager;
+import com.automationexercise.Pages.CartPage;
+import com.automationexercise.Pages.HomePage;
+import com.automationexercise.Pages.ProductsPage;
+import com.automationexercise.Utilities.DataUtils;
+import com.automationexercise.Utilities.Utility;
+import org.testng.asserts.SoftAssert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class AddtocartTest {
+
+
+    @BeforeMethod
+    public void setup(){
+        DriverFactory.createInstance(DataUtils.getEnvironmentPropertyValue("BROWSER"));
+        Utility.openWebsite(DataUtils.getEnvironmentPropertyValue("BASE_URL"));
+    }
+    @Test
+    public void TC01_addToCart(){
+        new SoftAssert().assertTrue(new HomePage().verifyHomePageVisability());
+        new CartPage().productsNavbarClick();
+        new ProductsPage()
+                .hoverOnBlueTopItemElement()
+                .bluetopddtoCartclick()
+                .continueShoppingButtonClick();
+        new ProductsPage()
+                .hoverOnmenTshirtItemItemElement()
+                .menTshirtAddtocartClick()
+                .continueShoppingButtonClick()
+                .cartNavbarElementClick();
+        new SoftAssert().assertTrue(new CartPage().isBlueTopWomenItemExists());
```

Comment:
How do you assert that element exists after already clicking on it? It should be before clicking, also you can make them **Hard Assertion** because if the element is not existing you can't click on so you needn't continue execution.